components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2507,6 +2509,7 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         const user = await this.checkAndBlockUser("updateOrgSettings");
         traceAPIParams(ctx, { orgId, userId: user.id });
         await this.guardTeamOperation(orgId, "update");
+        // TODO: call ImageBuilder ResolveBaseImage to dry test if we can access this image
```

Comment:
This is a very valid point. When do you think we should add this? /cc @akosyakov

Reply:
It's hard to have this valid point -> we can't tell if it's not exit or private image only, and even we have, it seems we don't want to have `dedicated` specify UX changes? cc @gtsiolis since [[internal chat](https://gitpod.slack.com/archives/C05H5UQBW6Q/p1694787506276619?thread_ts=1694785569.685029&cid=C05H5UQBW6Q)]

Reply:
**thought**: This could be out of the scope of this PR, but from the [relevant discussion](https://gitpod.slack.com/archives/C05H5UQBW6Q/p1695028961481619?thread_ts=1695026341.413039&cid=C05H5UQBW6Q) (internal) with @akosyakov, an idea could be to try to validate the image on load, and nudge org owners when something is wrong. This could be surfacing also in the dashboard as global alert. 💡 

| Checking | Error | OK |
|-|-|-|
| <img width="680" alt="Frame 1882" src="https://github.com/gitpod-io/gitpod/assets/120486/7d85fe67-1ddb-4904-8625-676064956d23"> | <img width="680" alt="Frame 1883" src="https://github.com/gitpod-io/gitpod/assets/120486/8175e3d6-811c-4d91-99f7-ed613a37386e"> | <img width="680" alt="Frame 1884" src="https://github.com/gitpod-io/gitpod/assets/120486/cae54ef9-45c9-4d81-85f8-871edec92ba8"> |

Later on, we could improve UX by also moving the help text and also this status check above the text input.
| Proposed UX for help text placement, out of the scope of this PR |
|-|
| <img width="680" alt="Frame 1881" src="https://github.com/gitpod-io/gitpod/assets/120486/363d94e7-348c-47a3-999c-00ea71cffe1e"> |

Reply:
yeah, something like that, async, and that it can catch issues with external configuration later too, i.e. misconfigured auth

We could consider adding it with next PR though?

Reply:
> We could consider adding it with next PR though?

+1

Reply:
let's remove TODo here and create a linear issue instead with George's design